_data/destinations/reference/microsoft-azure.yml

```diff
@@ -185,7 +185,7 @@ attributes:
     description: |
       {{ destination.display_name }} destinations do not have native support for Primary Keys.
 
-      To ensure existing rows can be updated in incrementally-replicated tables, Stitch will create an `sdc_primary_keys` table in every integration schema. This table contains the Primary Key data for every table in the given schema, which Stitch will use to de-dupe rows during loading.
+      To ensure existing rows can be updated in incrementally-replicated tables, Stitch will create an `_sdc_primary_keys` table in every integration schema. This table contains the Primary Key data for every table in the given schema, which Stitch will use to de-dupe rows during loading.
```

Comment:
And for every other `_sdc_*` hereafter:

I thought we determined that object names couldn't begin with underscores?

Reply:
I saw you had that in the table names section too and wasn't sure what we do if the customer had a table starting with underscore. At least for our reject and ok table they definitely start with an underscore.